libs/components/code-examples/src/lib/modules/text-editor/text-editor/example.component.html

```diff
@@ -1,20 +1,18 @@
-<div class="sky-padding-even-xl">
```

Comment:
Removing these wrappers so that the demos don't have excessive padding on the docs site.